Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/OnDemand/Utils/Common.hs

```diff
@@ -462,6 +462,7 @@ mkFulfillmentV2 mbDriver mbDriverStats ride booking mbVehicle mbImage mbTags mbP
                                 imageWidth = Nothing
                               },
                         personName = mbDInfo >>= Just . (.name),
+                        personGender = Nothing,
```

Comment:
Why is it hardcoded to Nothing ?

Reply:
Added nothing where ever gender value is not required